### PR TITLE
Delete images pushed to OCIR at the end of testing

### PR DESCRIPTION
**Pull request by rjeberhard**

### CODE REVIEW

File: new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java [hunk 1]
```diff
@@ -142,6 +147,61 @@ public void close() {
       deleteImage(operatorImage);
     }
 
+    // delete images from OCIR, if necessary
+    if (REPO_NAME.contains("ocir.io")) {
+      String token = getOcirToken();
+      if (token != null) {
+        if (miiBasicImage != null) {
+          deleteImageOcir(token, miiBasicImage);
```
Comment (sankarpn) [inline new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java:155]: The latest changes in develop branch seems to build more images in this class for ItDomainInImageWdt.java. Are we going to delete all those images?

**Comment by vanajamukkara** [comment]
@rjeberhard I see these messages, is it deleting the images?
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-model-in-image-tests-10/180/console

```
<05-26-2020 21:06:15> <INFO> <oracle.weblogic.kubernetes.extensions.ImageBuilders deleteImageOcir> <About to invoke: curl -skL -X "DELETE" -H "Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1OTA1MzA3NzUsIm5iZiI6MTU5MDUyNzE3NSwiaWF0IjoxNTkwNTI3MTc1LCJzdWIiOiJ7XCJ1c2VySWRcIjpcIm9jaWQxLnVzZXIub2MxLi5hYWFhYWFhYXd3bjU0ZmlqeHg1ZXJvbXBrZGo3b3J5dGhzY3E3eXBpaWtkYWJmbnF5bHl6aDRjYnptcmFcIixcInRlbmFudElkXCI6XCJvY2lkMS50ZW5hbmN5Lm9jMS4uYWFhYWFhYWFobWNiYjVtcDJoNnRvaDR2ajdheDUyNnh0bWlocm5lb3VteWF0NTU3cnZsb2xzeDYzaW1xXCIsXCJjbGFpbXNcIjpbXX0ifQ.FcM9FOv44ayAHEkQf8JYgWdoRliQO1N03r-_WX-RJEM" "https://****/20180419/docker/images/weblogick8s/mii-basic-image/2020-05-26-1590523182665">
{"errors":[{"code":"NAME_UNKNOWN","message":"Repository Name Unknown","detail":{"userSuppliedPath":"weblogick8s/mii-basic-image/2020-05-26-1590523182665"}}]}<05-26-2020 21:06:15> <INFO> <oracle.weblogic.kubernetes.extensions.ImageBuilders deleteImageOcir> <result.stdout: 
{"errors":[{"code":"NAME_UNKNOWN","message":"Repository Name Unknown","detail":{"userSuppliedPath":"weblogick8s/mii-basic-image/2020-05-26-1590523182665"}}]}>
<05-26-2020 21:06:15> <INFO> <oracle.weblogic.kubernetes.extensions.ImageBuilders deleteImageOcir> <result.stderr: 
>
```

**Comment by rjeberhard** [comment]
@vanajamukkara, in your run, the images don't exist to be deleted. There must have been an earlier failure to push them. Here's a run where the deletion is successful: https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-model-in-image-tests-10/169/console.

I checked the tag from your job and that tag is not present in OCIR.

**Comment by vanajamukkara** [comment]
oh yeah, I see it ...

**Pull request merged**